pkg/verifier/notation/truststore.go

```diff
@@ -122,3 +127,27 @@ func (s *trustStore) filterValidCerts(certs []*x509.Certificate) ([]*x509.Certif
 	}
 	return filteredCerts, nil
 }
+
+// parseErrFromKmpAndCertStore returns the error from KMP or CertStore based on the priority.
```

Comment:
Hi binbin, can you add some additional comments on why reconcile error have higher pri?

Reply:
Added comments on reconcile error, please let me know if it makes sense to you.